authentik/enterprise/audit/apps.py

```diff
@@ -0,0 +1,19 @@
+"""Enterprise app config"""
+from django.conf import settings
+
+from authentik.enterprise.apps import EnterpriseConfig
+
+
+class AuthentikEnterpriseAuditConfig(EnterpriseConfig):
+    """Enterprise app config"""
+
+    name = "authentik.enterprise.audit"
+    label = "authentik_enterprise_audit"
+    verbose_name = "authentik Enterprise.Audit"
+    default = True
+
+    def reconcile_install_middleware(self):
```

Comment:
```suggestion
    def reconcile_global_install_middleware(self):
```

Reply:
By always install the middleware, I meant adding it to settings.py (or apps.py which should overload it with the logic in settings.py)

Reply:
adding it to middleware in settings doesn't overwridte the main middleware list but rather just appends, so we have to do this here

Reply:
Ah yes, indeed, I missed that